go/cmd/dolt/commands/checkout.go

```diff
@@ -17,6 +17,8 @@ package commands
 import (
 	"context"
 
+	ref2 "github.com/liquidata-inc/dolt/go/libraries/doltcore/ref"
```

Comment:
What is wrong with you

Reply:
fixed.

Reply:
I mean I fixed the import.  What is wrong with me can't be fixed.